### ISSUE TITLE
How to use the control points for Reverse sync testing specified in "802.1AS Recovered Clock Quality Testing" ? 

### ISSUE THREAD

**Issue by MNaveen9229** [Avnu/gptp#84]
Hi All,

For the recovered clock testing using "reverse sync" method , it is specified in the "802.1AS Recovered Clock Quality Testing" document to use different control points as follows,

    A control point to enable and disable the Reverse Sync/Follow-up messages . The default value of this is FALSE.
    A control point to set the domain of Reverse Sync/Follow-up messages . For certification it is proposed that a different unique domain should be used than the one that’s being measured. When using in deployed networks a different domain shall be used.
    A control point to set the Reverse Sync/Follow-up message rate as % of incoming rate . The default is defined in Section 6.3.

Here does "reverseSyncEnabled","reverseSyncDomain" and "reverseSyncRate" refers to the control variables which has to be newly created , and based on this variables the code has to be updated ?
(or)
Does these control points are already available in the opensource code. I have searched for this variables in the code, but was unable to find them.

Any suggestions/ references to how to proceed on this will be really helpful.

Also kindly recommend where i can find reference code for ingress/reverse sync testing.

**Comment by zeerd** [comment]
No. I don't think Avnu has opened those testing source codes, at least not here.
